TSG/LCM/Add-Node-Failed-at-SetTrustedHostsOnNewNode-Step.md

```diff
@@ -0,0 +1,165 @@
+# How to Resume Add Node operation on BrownField Environment failed at SetTrustedHostsOnNewNode Step
+# Description 
+
+This article describes how to resume Add node operation on a brownfield upgraded cluster when failed at SetTrustedHostsOnNewNode.
+# Symptoms
```

Comment:
Add here that this is seen only for 2408 build and will be fixed in upcoming release?